meson.build

```diff
@@ -1568,8 +1574,11 @@ conf.set10('HAVE_ZLIB', have)
 
 want_bzip2 = get_option('bzip2')
 if want_bzip2 != 'false' and not skip_deps
-        libbzip2 = cc.find_library('bz2',
-                                   required : want_bzip2 == 'true')
+        libbzip2 = dependency('bzip2', required : false)
+        if not libbzip2.found()
+                # Debian and Ubuntu do not provide the .pc file.
+                libbzip2 = cc.find_library('bz2', required : want_bzip2 == 'true')
+        endif
```

Comment:
There is no .pc file for Debian/Ubuntu to not-provide. bzip2 doesn't have one and never has. Although I did submit a patch to add one: https://sourceware.org/pipermail/bzip2-devel/2022q2/000169.html

Reply:
Interesting. Maybe, fedora picks your patch.